### ISSUE TITLE
Compact Mode

### ISSUE THREAD

**Issue by royto** [royto/logbook-card#135]
Created from #134 

A compact mode would also be great perhaps where the start/end times are hidden and the spacing between entries is minimized, this would allow some nice visual customization

**Comment by royto** [comment]
Hello @sharkpunch5, is something like this is better for you ?
![image](https://github.com/user-attachments/assets/d39f8263-caf7-4b29-9eeb-9a6e9f5cbee6)

**Comment by sharkpunch5** [comment]
Awesome thankyou, looks great, tried playing with card mod and turning off the start/end time to achieve similar with the below but yours is much cleaner.

```card_mod:
  style: |
    .item.history {
      height: 35px; /* Adjust to desired height */
      padding: 5px; /* Optional padding */
    }```
    
The ideal scenario is that I can use the relative time or start time to replace the duration, here is what i achieved without that ideally that start time for the action is there instead of duration, in practice, knowing that the door was closed for 14 hours overnight is not super useful info, what I'd like to know is when it was last closed or opened. Perhaps that is also a separate feature request again but also part of moving data with compact mode.

![image](https://github.com/user-attachments/assets/498ff860-d81b-474c-a4db-bac9a3d789da)

**Comment by royto** [comment]
Can you try with this custom style

```
type: custom:logbook-card
entity: sun.sun
show:
  state: true
  duration: false
  start_date: true
  end_date: false
  icon: true
  separator: true
  entity_name: true
card_mod:
  style: |
    .item-content {
      display: grid;
      grid-template: "state date" auto / 1fr auto;
    }
    .state {
        grid-area: "state";
    }
    
    .date {
      grid-area: "date"
      font-size: 0.85rem;
      font-style: italic;
    }
``` 

It should render like this

![image](https://github.com/user-attachments/assets/3f491551-1d1d-434d-bbaf-d05f37f296b3)

**Comment by sharkpunch5** [comment]
looks simlar however it appears to be rendering the height of item-content with the extra space in my browser?
![image](https://github.com/user-attachments/assets/5be9445e-39ec-4c38-bac6-bbe8ee907848)

**Comment by sharkpunch5** [comment]
actually using this i was able to get something like this on my card

```
  style: |
    .item.history {
      height: 25px;
      padding: 5px; 
    }
    .item-content {
      display: grid;
      grid-template: "state date" auto / 1fr auto;
    }
    .state {
        grid-area: "state";
    }
    
    .date {
      grid-area: "date"
      font-size: 0.85rem;
      font-style: italic;
    }
```
![image](https://github.com/user-attachments/assets/e689702a-ecc1-4dce-bb49-3bffe5623864)

looks great thankyou :)

**Comment by royto** [comment]
the new released [2.5.6](https://github.com/royto/logbook-card/releases/tag/2.5.6) fixes the extra space 😉
